layouts/mdx.tsx

```diff
@@ -31,11 +33,19 @@ interface MDXLayoutProps {
 
 export default function MDXLayout(props: MDXLayoutProps) {
   const { frontmatter, children } = props
-  const routes = getRoutes(frontmatter.slug)
+  // const routes = getRoutes(frontmatter.slug)
 
+  // const route = findRouteByPath(removeFromLast(frontmatter.slug, '#'), routes)
+  // const routeContext = getRouteContext(route, routes)
+  const routes = getRoutes(frontmatter.slug)
+  console.log('routes', routes)
+  console.log('removeLast', removeFromLast(frontmatter.slug, '#'))
   const route = findRouteByPath(removeFromLast(frontmatter.slug, '#'), routes)
+  console.log('route', route)
   const routeContext = getRouteContext(route, routes)
 
+  console.log('routeContext', routeContext)
```

Comment:
Let's clean up the unused commented code and console logs.

Reply:
Good catch ! Will do :)